synapse/storage/devices.py

```diff
@@ -127,16 +133,26 @@ def get_devices_by_remote(self, destination, from_stream_id, limit):
         # (user_id, device_id) entries into a map, with the value being
         # the max stream_id across each set of duplicate entries
         #
-        # maps (user_id, device_id) -> stream_id
+        # maps (user_id, device_id) -> (stream_id, context)
         # as long as their stream_id does not match that of the last row
```

Comment:
Still needs a review from @anoadragon453

Reply:
Resolved after IRL?

Reply:
Will be updated in another pr